src/gen-utils.ts

```diff
@@ -198,6 +198,67 @@ export function createGlowElement(options: TextGlowProps, defaults: TextGlowProp
 	return strXml
 }
 
+/**
+ * Create gradient elements i.e. `a:gsLst` and `a:lin`
+ * @param {Gradient} gradient
+ * @param {string} internalElements
+ * @see http://officeopenxml.com/drwSp-GradFill.php
+ * @returns XML string
+ */
+function createGradientElements(gradient: Gradient, internalElements: string): string {
+	let strXml = '',
```

Comment:
Not functional at all, but consistent with the style used in the project, see `createGlowElement` above